share/ext/tremolo/build_tremolo.sh

```diff
@@ -13,84 +13,56 @@
 # CONDITIONS OF ANY KIND, either express or implied. See the License for the
 # specific language governing permissions and limitations under the License.
 
+readonly REPO_URL=https://github.com/defold/defold-tremolo.git
```

Comment:
Added a new library to more easily keep control of updates and our changes to the library.